src/layman/layer/geoserver/tasks.py

```diff
@@ -59,9 +62,19 @@ def refresh_wms(
                                               layername,
                                               description,
                                               title,
-                                              access_rights,
                                               geoserver_workspace=geoserver_workspace,
                                               )
+    elif file_type == settings.FILE_TYPE_RASTER:
+        gs_util.delete_wms_layer(geoserver_workspace, layername, settings.LAYMAN_GS_AUTH)
+        gs_util.delete_wms_store(geoserver_workspace, settings.LAYMAN_GS_AUTH, wms.get_qgis_store_name(layername))
+        gs_util.delete_feature_type(geoserver_workspace, layername, settings.LAYMAN_GS_AUTH)
```

Comment:
I believe these three lines are extra and should be removed, because if we do PATCH with `file`, feature type, cascading WMS store, and cascading WMS layer are deleted synchronously.